js/utils.js

```diff
@@ -458,6 +458,130 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     }
   }
 
+  function getDateModifiedValues(options) {
+    switch (options.filterModifier) {
+      case 'today':
+        return {
+          today: moment().format('DD/MM/YYYY'),
+          formatedDate: moment(options.date).format('DD/MM/YYYY')
```

Comment:
I don't understand what the requirement is to format it into a string, and more important, why this format.

Reply:
@tonytlwu.

When searching for how to correctly compare dates using the moment js, I found more than one option for comparing strings, which I was surprised myself, but it works correctly.
The use of formats was necessary in order to correctly compare dates with an ordered time shift.
For example:
the record in the table is equal to 26 April 2021 16:30
our current date together with the time of April 26, 2021 12:35

to use the shift by the hour, I need to disregard the minutes, as I understood because there will be no coincidence, since the minutes differ, based on this, the Date and check check will almost always be false. But I can also be mistaken in that I have gone too deep into the intricacies of checking the time.

Reply:
1. You can use the functions available in moment for comparison. See https://momentjs.com/docs/#/query/
2. If you want to ignore minutes, then set the minutes to 0 rather than ignoring it through string comparison

For example:

```
moment().startOf('hour') // This sets everything that's less than an hour to 0
```

Reply:
@tonytlwu, okay i'll finish all your comments due next review.